signer/src/main.rs

```diff
@@ -382,5 +382,5 @@ async fn run_transaction_coordinator(ctx: impl Context) -> Result<(), Error> {
         dkg_max_duration: Duration::from_secs(10),
     };
 
-    coord.run().await
```

Comment:
I think we should make this configurable, because if we find that we have significant issues somehow when there's a Bitcoin block storm it would be good to be able to tweak this. 

I don't think it'll be a hard change to make; the config classes are pretty clear, and you can put it under the bitcoin config settings [here](https://github.com/stacks-network/sbtc/blob/main/signer/src/config/mod.rs#L101) named something like `bitcoin_processing_delay`, and you'll have access to it via the `ctx` variable.

Reply:
I like that. The reason I didn't do it already is that there seem to be a bunch of hard-coded values already, eg. here: https://github.com/stacks-network/sbtc/blob/c8c133d0d2e5cd8c572bb964c03ba0b25a1febef/signer/src/main.rs#L380.

Let me add it.

Reply:
Sounds good, but I don't think this is a value that we want signers freely messing around with, so we should be careful about how we handle it in the `default.toml` and config examples.

Reply:
Resolved.